app/assets/javascripts/course/assessment/submission/answer/answer.js

```diff
@@ -61,6 +61,30 @@
   }
 
   /**
+   * Shows reply buttons for annotations.
+   *
+   * `addBack` is required since the AJAX response inserts the reply button footer separately
+   * rather than as part of the main discussion topic. `find` will fail to select the button if
+   * the button is `element` itself.
+   *
+   * @param element
+   */
+  function showReplyButton(element) {
+    var REPLY_COMMENT_SELECTOR = '.reply-annotation';
+    $(element).find(REPLY_COMMENT_SELECTOR).addBack(REPLY_COMMENT_SELECTOR).show();
```

Comment:
Since we have already narrowed the search to be within `element`, do we still need the `DOCUMENT_SELECTOR`?